packages/crypto/test/messageEncryption.test.ts

```diff
@@ -213,4 +222,39 @@ describe('message encryption', () => {
 		//	)
 		//})
 	})
+
+	it('should convert to a valid babylon private key', async () => {
```

Comment:
This test doesn't seem relevant to the issue at hand.

Even the old toString would have satisfied this test.

A better test is that `babylonKey.toString() === olympiaKey.toString()`

Reply:
this seemed to me to be the root issue? the test also did fail with the old method. But sure, I can update it.